build-logic/idea/src/main/kotlin/gradlebuild.ide.gradle.kts

```diff
@@ -87,13 +90,36 @@ fun ProjectSettings.configureRunConfigurations() {
     }
 }
 
+fun ProjectSettings.configureTaskTriggers() {
+    taskTriggers {
+        project(":internal-integ-testing").afterEvaluate {
+            afterSync(tasks.named("generateLanguageAnnotations"))
+        }
+    }
+}
+
+tasks.register("printIdeaSettings") {
```

Comment:
This looks a lot like something that is just adding "garbage" to our build. No one will use it since no one knows about this. We spend a lot of time removing things like this recently because they just added noise to the build.

I am not sure what the solution is. But if we can, I would like to avoid having this in the build logic. Maybe we can just document the approach somewhere?

If you feel like this is really needed, could you move it to a separate PR to discuss with the team how to deal wit such things? 

:x: Also: Never use `doLast()` but implement tasks as classes.

Reply:
I can remove this